src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,34 @@
+package seedu.addressbook.data;
+
+import java.util.ArrayList;
+
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.tag.Tag;
+
```

Comment:
Please include a header comment to indicate what this interface/class/method does?